packages/example/components/ApiForm/ApiCombobox.tsx

```
@@ -0,0 +1,164 @@
+  return <div className='flex flex-col'>
+    {label && (
+      <Label htmlFor={id}>
+        {label}
+        {required && <span className="text-red-500">*</span>}
+      </Label>
+    )}
+    <Popover open={open} onOpenChange={setOpen}>
+      <PopoverTrigger asChild>
+        <Button
+          variant="outline"
+          role="combobox"
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)